src/components/Onboarding/kit/FileUploaderField/index.js

```diff
@@ -190,17 +202,7 @@ export const FileUploaderField = ({
                     `}
                   />
                 </div>
-                {validExtensions && (
-                  <div
-                    css={`
-                      margin-top: ${1 * GU}px;
-                      color: ${mainColor};
-                      font-weight: bold;
-                    `}
-                  >
-                    Valid file format: {toOxford(validExtensions)}
-                  </div>
-                )}
+                <TextFileUploader />
```

Comment:
What do you think of adding a new prop instead? That allows configuring if we will show the valid extensions or not.

Reply:
We have no case on the designs where we are doing that so i don't think is necessary  at all